src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SpeechToTextImplementation.windows.cs

```diff
@@ -86,7 +86,6 @@ async Task<string> ListenOnline(CultureInfo culture, IProgress<string>? recognit
 		await using (cancellationToken.Register(async () =>
 		{
 			await StopRecording();
-			speechRecognitionTaskCompletionSource.SetCanceled();
```

Comment:
How come we're no longer canceling the `TaskCompletionSource`?

Reply:
When we call StopListening, Listener Completed event is raised and set TaskCompletionSource

Reply:
I think we should add methods Start/Stop/Cancel for better control. Then using events we just return results. It should be more understandable then using IProgress

Reply:
But it’s for the next PR.